lightningd/options.c

```diff
@@ -66,6 +66,23 @@ static void tal_freefn(void *ptr)
 #define TIME_FROM_MSEC(msec) \
 	{ { .tv_nsec = ((msec) % 1000) * 1000000, .tv_sec = (msec) / 1000 } }
 
+static char *opt_set_u64(const char *arg, u64 *u)
+{
+	char *endp;
+	unsigned long l;
+
+	assert(arg != NULL);
+
+	/* This is how the manpage says to do it.  Yech. */
+	errno = 0;
+	l = strtoul(arg, &endp, 0);
```

Comment:
Technically this needs to be `strtoull`, though I don't think we need u64 for timeouts anyway.  But in case anyone uses this in future...

Reply:
Okay, will fix.